oceanwaters/doc/setup_dev_env.md

```diff
@@ -112,8 +113,48 @@ make universalExec plexil-compiler checkpoint
 OceanWATERS.  Additional build information is available
 [here](http://plexil.sourceforge.net/wiki/index.php/Installation).
 
-* Also note that the latest version of this branch tested with OceanWATERS is tagged `OceanWATERS-v7.1`.  
+* Also note that the latest version of this branch tested with OceanWATERS is tagged `OceanWATERS-v7.1`. 
+
+### GSAP
+
+The OceanWATERS distribution includes a power system  module (`ow_power_system`) that at present
+depends on GSAP, an open-source battery prognostics executive. GSAP  must be installed
+*prior* to building the `ow_power_system` package.
+
+
+* Check out the source code:
+```
+git clone https://github.com/nasa/GSAP.git gsap
+```
+
+The default git branch is v2.0.0, which is the latest stable version
```

Comment:
The default branch of GSAP is actually 'master'.  There is a branch 'release_2_0_0'.  OPlease clarify which one we should use.  If it's the latter, the '--branch release_2_0_0' option to 'git clone' is a good way to do it, as suggested above.

.

Reply:
updated to utilize release_2_0_0 now so that any future updates to the master branch of GSAP don't break our code